### PR TITLE
Add experimental flag for the blurred placeholders

### PR DESCRIPTION
**Pull request by addieachan**
Partial for #15146.

- Creates the experimental flag for blurry image placeholders.

### CODE REVIEW

File: builtins/amp-img.js [hunk 1]
```diff
@@ -45,8 +48,15 @@ export class AmpImg extends BaseElement {
     /** @private {?UnlistenDef} */
     this.unlistenLoad_ = null;
 
+    /** @private @const {boolean} */
+    this.useNativeSrcset_ = isExperimentOn(this.win, 'amp-img-native-srcset');
```
Comment (choumx) [inline builtins/amp-img.js]: This looks like a mistake.

File: builtins/amp-img.js [hunk 2]
```diff
@@ -45,8 +48,15 @@ export class AmpImg extends BaseElement {
...
+
+    /** @private @const {boolean} */
+    this.expOn_ = isExperimentOn(this.win, 'amp-img-blurry-placeholder');
```
Comment (choumx) [inline builtins/amp-img.js]: No need to check this yet.

File: tools/experiments/experiments.js [hunk 3]
```diff
@@ -309,6 +309,11 @@ const EXPERIMENTS = [
     spec: 'https://github.com/ampproject/amphtml/issues/14740',
     cleanupissue: 'https://github.com/ampproject/amphtml/issues/16075',
   },
+  {
+    id: 'amp-img-blurry-placeholder',
```
Comment (choumx) [inline tools/experiments/experiments.js]: "amp" is redundant. Let's just call this 'blurry-placeholder'.

File: tools/experiments/experiments.js [hunk 4]
```diff
@@ -309,6 +309,11 @@ const EXPERIMENTS = [
...
+    name: 'Enables a blurry image placeholder as an amp-img loads',
+    spec: 'https://github.com/ampproject/amphtml/issues/15146',
```
Comment (choumx) [inline tools/experiments/experiments.js]: File a new issue for cleaning up this experiment and add it to `cleanupissue` key here.

File: builtins/amp-img.js (hunk 2 above)
Comment (addieachan) [reply builtins/amp-img.js]: Sorry for the confusion, but by "check this" I'm assuming you mean I don't have to initialize this yet. If I'm wrong, again really sorry, just please let me know

File: tools/experiments/experiments.js (hunk 4 above)
Comment (addieachan) [reply tools/experiments/experiments.js]: Filed an issue, but I don't have access to add labels. Sorry if I'm wrong, but if you can add it/tell me how that would be appreciated.

**Comment by addieachan** [comment]
@gmajoulet please feel free to add yourself/anyone who you think should see it review this!

**Comment by addieachan** [comment]
If one of you could help me merge this (assuming you don't need other changes from me), that would be great!

**Pull request merged**